src/uct/ib/dc/dc_mlx5.h

```diff
@@ -247,6 +248,25 @@ typedef struct uct_dc_mlx5_ep_fc_entry {
 
 KHASH_MAP_INIT_INT64(uct_dc_mlx5_fc_hash, uct_dc_mlx5_ep_fc_entry_t);
 
+typedef enum uct_dc_mlx5_ep_type {
+    UCT_DC_MLX5_EP_TYPE_DEFAULT,
+    UCT_DC_MLX5_EP_TYPE_KEEPALIVE,
+    UCT_DC_MLX5_EP_TYPE_FC
+} UCS_S_PACKED uct_dc_mlx5_ep_type_t;
+
+typedef union uct_dc_mlx5_dci_config {
+    struct {
+        uint8_t sl;
+        uint8_t port_affinity;
+        uint8_t path_index;
+        uint8_t ep_type;
+        uint8_t max_rd_atomic;
```

Comment:
should be a single bit because it can be either 16 or 64, maybe we can unite it with ep_type and port_affinity?